src/controllers/auth.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express";
+import { PrismaClient } from "@prisma/client";
+import bcryptjs from "bcryptjs";
+import jwt from "jsonwebtoken";
+
+const prisma = new PrismaClient();
```

Comment:
you can separate this into a different file `db.ts` and handle all the connection for the db there.